main.go

```diff
@@ -178,7 +182,7 @@ func run() int {
 		sc.Lock()
 		conf := sc.C
 		sc.Unlock()
-		prober.Handler(w, r, conf, logger, rh, *timeoutOffset, nil)
+		prober.Handler(w, r, conf, logger, rh, *timeoutOffset, nil, &moduleUnknownCounter)
```

Comment:
`prometheus.Counter` is an interface, so I'm pretty sure we don't need to pass a reference.

CC @roidelapluie